src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -857,7 +855,31 @@ public ValueTask DisposeAsync()
     private static bool IsHttpsOrNull(BindingAddress? address) => address == null || string.Equals(address.Scheme, "https", StringComparison.Ordinal);
 }
 
-public record EndpointInfo(string Address, IPEndPoint EndPoint, bool IsHttps);
+public record EndpointInfo(BindingAddress BindingAddress, IPEndPoint EndPoint, bool IsHttps)
+{
+    public string GetResolvedAddress(bool replaceIPAnyWithLocalhost = false)
+    {
+        if (!IsAnyIPHost(BindingAddress.Host))
+        {
+            return BindingAddress.Scheme.ToLowerInvariant() + Uri.SchemeDelimiter + BindingAddress.Host.ToLowerInvariant() + ":" + EndPoint.Port.ToString(CultureInfo.InvariantCulture);
```

Comment:
Pretty sure `Uri`'s `Scheme` and `Host` are always lowercase.

Reply:
This is a `BindingAddress`, an ASP.NET Core thing.